app/navigation/Navigator.scala

```diff
@@ -52,10 +52,15 @@ class Navigator @Inject()() extends Logging {
   }
 
   def claimJourneyFlow(userAnswers: UserAnswers): Call = {
-    userAnswers.get(ClaimedForTaxYear2020) match {
-      case Some(claimedAlready) if claimedAlready   => routes.DisclaimerController.onPageLoad()
-      case Some(claimedAlready) if !claimedAlready  => routes.SelectTaxYearsToClaimForController.onPageLoad()
-      case None                                     => routes.IndexController.onPageLoad()
+    userAnswers.get(HasSelfAssessmentEnrolment) match {
+      case None         => routes.IndexController.onPageLoad()
```

Comment:
Do we need this?

Reply:
We only need it to stop the warning "match is not exhaustive!”, and it's there just in case.